applications/ShapeOptimizationApplication/custom_utilities/response_functions/am_boundary_slope_angle_response_function_utility.cpp

```diff
@@ -0,0 +1,124 @@
+// ==============================================================================
+//  KratosShapeOptimizationApplication
+//
+//  License:         BSD License
+//                   license: ShapeOptimizationApplication/license.txt
+//
+//  Main authors:    Reza Najian Asl
+//
+// ==============================================================================
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "custom_utilities/response_functions/am_boundary_slope_angle_response_function_utility.h"
+#include "shape_optimization_application.h"
+#include "utilities/variable_utils.h"
+
+// ==============================================================================
+
+namespace Kratos
+{
+
+AMBoundarySlopeAngleResponseFunctionUtility::AMBoundarySlopeAngleResponseFunctionUtility(ModelPart& rModelPart, Parameters ResponseSettings)
```

Comment:
browsing through this seems very similar to the other class

maybe you can refactor/unify?